lib/processes.js

```diff
@@ -103,17 +103,32 @@ function launchAllProcesses(config) {
       if (chromePath) {
         config.set('desiredCapabilities.chromeOptions.binary', chromePath);
       }
-      var args = config.get('desiredCapabilities.chromeOptions.args', [
+
+      var args = [
         '--disable-application-cache',
         '--media-cache-size=1',
         '--disk-cache-size=1',
         '--disk-cache-dir=/dev/null',
         '--disable-cache',
         '--disable-desktop-notifications'
-      ]);
+      ];
+
       if (config.get('chrome.headless', !process.env.DISPLAY)) {
         args.push('--headless');
       }
+
+      if (config.has('desiredCapabilities.chromeOptions.args')) {
+        if (config.has('desiredCapabilities.chromeOptions.merge')
+          && config.get('desiredCapabilities.chromeOptions.merge') === false) {
+          args = config.get('desiredCapabilities.chromeOptions.args');
+        } else {
+          args = args.concat(config.get('desiredCapabilities.chromeOptions.args'))
+            .filter(function filterIt(opt, idx, arr) {
+              return arr.indexOf(opt) === idx;
+            });
```

Comment:
I think this might need to be a bit more sophisticated for cases like `--disk-cache-size=2`

Reply:
yeah that is a good point I forgot about options like `--disk-cache-size=2`